File: src/main/java/trap/model/RoundScore.java
package trap.model;

import lombok.Builder;

@Builder
public record RoundScore(
        int eventId,
        String event,
        int locationId,
        String location,
        String eventDate,
        String squadName,
        String team,
        String athlete,
        String classification,
        String gender,
        int round1,
        int round2,
        int round3,
        int round4,
        int round5,
        int round6,
        int round7,
        int round8,
        String type
) {
}

File: src/main/java/trap/model/RoundScore.kt
package trap.model

class RoundScore(val eventId: Int,
                 val event: String?,
                 val locationId: Int,
                 val location: String?,
                 val eventDate: String?,
                 val squadName: String?,
                 val team: String?,
                 val athlete: String?,
                 val classification: String?,
                 val gender: String?,
                 val round1: Int,
                 val round2: Int,
                 val round3: Int,
                 val round4: Int,
                 val round5: Int,
                 val round6: Int,
                 val round7: Int,
                 val round8: Int,
                 val type: String) {
    val uniqueName: String
        get() = "$athlete $team $classification $type"
}
